config/routes.rb

```diff
@@ -204,6 +204,7 @@
         resources :organizations, only: [] do
           collection do
             get "/:org_username", to: "organizations#show"
+            get "/:org_username/users", to: "organizations#users"
           end
         end
```

Comment:
do you think we could use resource routes here?

For example:

```ruby
        resources :organizations, only: [:show], param: :username do
          resources :users, only: [:index], to: "organizations#users"
        end
```

(renamed the parameter to `:username` as it's implicit it's part of an org)

this will also have the effect of generating helpers such as: `api_organization_users_path` and `api_organization_path` that you can use in the tests.

Also, if the `API::OrganizationsController` becomes too big we can easily move the actions to their own resource controller (for example: `API::OrganizationsUsersController` without changing the external URLs)

Reply:
Ya for sure. I just pushed up an update for the routes